### PR TITLE
Merge MCreator to MCToolkit

### PR DESCRIPTION
**Pull request by Max094Reikeb**

### CODE REVIEW

File: src/main/java/net/mcreator/ui/dialogs/TextureImportDialogs.java [hunk 1]
```diff
@@ -122,6 +123,41 @@ public static void importArmor(final MCreator fra) {
 			}
 	}
 
+	public static void importTexturesBlockOrItem(MCreator fr, BlockItemTextureSelector.TextureType type) {
```
Comment (Goldorion) [inline src/main/java/net/mcreator/ui/dialogs/TextureImportDialogs.java:126]: Revert changes of the file from this line

File: src/main/java/net/mcreator/ui/dialogs/BlockItemTextureSelector.java [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+public class BlockItemTextureSelector extends MCreatorDialog {
```
Comment (Goldorion) [inline src/main/java/net/mcreator/ui/dialogs/BlockItemTextureSelector.java:42]: Delete this file as it's now GeneralTextureSelector

**Review by Goldorion — CHANGES_REQUESTED** [review]
Please fix the following requested changes, so we won't have bugs.

File: src/main/java/net/mcreator/ui/dialogs/TextureImportDialogs.java (hunk 1 above)
Comment (Max094Reikeb) [reply src/main/java/net/mcreator/ui/dialogs/TextureImportDialogs.java]: Done

File: src/main/java/net/mcreator/ui/dialogs/BlockItemTextureSelector.java (hunk 2 above)
Comment (Max094Reikeb) [reply src/main/java/net/mcreator/ui/dialogs/BlockItemTextureSelector.java]: Done, I deleted the file

**Pull request merged**